lsp-completion.el

```diff
@@ -614,7 +611,12 @@ Returns resolved completion item details."
                                                              :_emacsStartPoint
                                                              (or (lsp-completion--guess-prefix item)
                                                                  bounds-start)))
-                                                  it))))
+                                                  it)
+                                            ;; remove items with no label or filterText
+                                            (-remove
```

Comment:
I wonder why we need to filter them out from here? shouldn't the filter function does that already at a later time?

Reply:
It doesn't. That to-internal function was a mistake